tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -8,13 +8,13 @@
   <ItemGroup>
     <!-- Integration: Microsoft.Data.SqlClient -->
     <!-- Assembly: Microsoft.Data.SqlClient -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Data.SqlClient/3.0.1 -->
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.0.1" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Data.SqlClient/4.0.0 -->
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="4.0.0" />
```

Comment:
We need to remove this file I think - these integrations haven't changed here, and it's misleading. We need to change the action

Reply:
Sorry, wasn't clear, I meant remove the file from the diff